src/rgw/rgw_file.h

```diff
@@ -1617,7 +1611,7 @@ class RGWReaddirRequest : public RGWLibRequest,
 
       lsubdout(cct, rgw, 15) << "RGWReaddirRequest "
 			     << __func__ << " "
-			     << "list uri=" << s->relative_uri << " "
```

Comment:
there was no error here

Reply:
no error, just a warning or "s" ambiguity because it also exist in one of the parents as "protected" member.
will do the same as in the other place (call the local variable "state")

Reply:
ok